Src/Plugins/Security/Utilities/FilteringHelpers.cs

```diff
@@ -48,5 +48,24 @@ public static bool LikelyPowershellVariable(string input)
 
             return true;
         }
+
+        public static bool PasswordIsInCommonVariableContext(string secret)
```

Comment:
>InCommo [](http://example.com/codeflow?start=37&length=7)

Can our rules in SPMI use this function as well after this is merged?

Reply:
Yes, the function I copied from your previous implementation. The other projects has a reference to Security can use it.